_drafts/clojure.org

```diff
@@ -0,0 +1,273 @@
+#+OPTIONS: toc:nil
+#+BEGIN_HTML
+---
+layout: post
+title: Criação de variáveis e contextos léxicos em Clojure
+tags:
+- coding
+- clojure
+---
+#+END_HTML
+
+Clojure é uma linguagem funcional, em que podemos tratar funções como
+qualquer outro primitivo da linguagem, passando como argumentos,
+retornos e salvando em estruturas de dados.
+
+Como funções são tão importantes, nós iremos criar diversas delas nos
+nossos programas.  Mas qual a diferença entre criar uma função com
+~(defn função [arg] arg)~ e ~(fn função [arg] arg)~ ?
```

Comment:
Eu rodei um regex pra arrumar os `função`s escritos errados e foi o codigo junto. Acabei deixando pq é permitido, mas posso mudar sim for muito confuso/esquisito/errado/etc